py-polars/Makefile

```diff
@@ -27,7 +28,7 @@ venv:
 	:
 else # use VENV
 	VENV_BIN    := $(abspath $(VENV_BIN))/# no space after trailing '/'
-	VENV_ACT_BIN = $(VENV_BIN)# no space after closing ')'
+	VENV_ACT_BIN = $(CONDA_DEACT) . $(VENV_BIN)activate; $(VENV_BIN)# no space after closing ')'
```

Comment:
We are activating the virtual environment for each time we call something (pytest, black, etc.). This is wasteful and doesn't make sense. We know where the executable is (in the $VENV folder), just call it directly. Plus the terminal becomes extremely messy now with all the full paths.

Reply:
This is how `make` operates [0]. Every line in the target recipe is run in its own sub-shell providing full task isolation.  Environment activation/deactivation happens only inside sub-shells. There is no downside to it.

[0] https://www.gnu.org/software/make/manual/make.html